site/package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "yarn build-blocks && yarn generate-blocks-data && yarn generate-sitemap && next build",
+    "build": "yarn build-blocks && yarn generate-blocks-data && yarn generate-sitemap && yarn db:indexes && next build",
     "build-block": "./scripts/build-blocks.sh",
     "build-blocks": "find ../hub -type f | xargs ./scripts/build-blocks.sh",
     "clean": "rimraf ./.next/",
-    "dev": "yarn generate-sitemap && yarn generate-blocks-data && next dev",
+    "db:indexes": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/db-indexes",
```

Comment:
This gives us a mix of `action:subject` and `subject:action` scripts. Ideally, we want to stick with the former for consistency. Not sure how exactly to change the naming at this point, but we might need to clean things up a little in a separate PR.

Reply:
In an ideal world, some of our scripts are hierarchical, so running `x` implies running `x:a` and `x:b`. In turn, `x:a` can involve `x:a:n` and `x:a:m`. We already do this for `lint:*` and `fix:*` scripts and can do so too in other scenarios.